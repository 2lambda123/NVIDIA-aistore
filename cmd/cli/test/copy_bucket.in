ais create bucket $BUCKET_1 $BUCKET_2

for d1 in {0..4}; do for d2 in {0..100}; do mkdir -p /tmp/rdir/test$d1/$d2 && echo -n $RANDOM > /tmp/rdir/test$d1/$d2/test.txt; done; done
ais put "/tmp/rdir/test{0..4}/{0..100}/test.txt" $BUCKET_1 -y // IGNORE

ais cp bucket aiss://$BUCKET_1/ $BUCKET_2/ // FAIL "Invalid bucket provider "aiss"."
ais cp bucket $BUCKET_1/ aiss://$BUCKET_2/ // FAIL "Invalid bucket provider "aiss"."
ais cp bucket $BUCKET_1 $BUCKET_2

ais wait xaction copybck $BUCKET_2
ais show xaction copybck $BUCKET_1
ais show xaction copybck
ais show xaction copybck $BUCKET_3 // FAIL "bucket "$BUCKET_3" does not exist"

rm -rf /tmp/rdir // IGNORE
