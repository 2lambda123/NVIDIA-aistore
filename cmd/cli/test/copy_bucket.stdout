"$BUCKET_1" bucket created
"$BUCKET_2" bucket created
Bucket copy operation ("$BUCKET_1" => "$BUCKET_2") is in progress.
To check the status, run: ais show xaction copybck $BUCKET_2
DAEMON ID	 ID	 KIND	 BUCKET	 OBJECTS	 BYTES	 START	 END	 ABORTED
DAEMON ID	 ID	 KIND	 BUCKET	 OBJECTS	 BYTES	 START	 END	 ABORTED
